inc/branding/namespace.php

```diff
@@ -77,9 +77,16 @@ function add_color_scheme() {
 
 /**
  * Enqueue the branding scripts and styles
+ *
+ * @param string $hook
  */
-function enqueue_admin_scripts() {
+function enqueue_admin_scripts( string $hook ) {
+
 	wp_enqueue_style( 'altis-branding', plugin_dir_url( dirname( __FILE__, 2 ) ) . 'assets/branding.css', [], '2019-04-24-1' );
+
+	if ( $hook === 'site-new.php' ) {
+		wp_enqueue_script( 'customize-settings', plugin_dir_url( dirname( __FILE__, 2 ) ) . 'assets/customize-settings.js', [], false, true );
```

Comment:
```suggestion
		wp_enqueue_script( 'altis-customize-settings', plugin_dir_url( dirname( __FILE__, 2 ) ) . 'assets/customize-settings.js', [], false, true );
```

Reply:
Actually also, this needs to be moved across to the right file, otherwise it'll break when branding is disabled.